### PR TITLE
Updated response-body of /v1/regard-application

### PR DESCRIPTION
**Pull request by sunil-kumar-meena**
Fixes: 
https://github.com/openBackhaul/ApplicationLayerTopology/issues/570

Note: Last PR https://github.com/openBackhaul/ApplicationLayerTopology/pull/574 got merged into develop branch, raising new PR again, also suggest do we have to revert the last changes from develop branch also?

reviewer @PrathibaJee

### CODE REVIEW

File: server/service/individualServices/regardApplicationAutomation.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ async function RequestForInquiringTopologyChangeInformationWithDefaultKey(applic
     let responseCode = response.status;
     if (!responseCode.toString().startsWith("2")) {
       result["successfully-connected"] = false;
-      result["reason-of-failure"] = `ALT_NOT_REACHABLE`;
+      result["reason-of-failure"] = `ALT_DID_NOT_REACH_NEW_APPLICATION`;
```
Comment (PrathibaJee) [inline server/service/individualServices/regardApplicationAutomation.js:109]: if (!responseCode.toString().startsWith("2")) , then you have to check whether the if responseCode.toString() == "408" to confirm that the application is actually not reachable. ( Note : Include the response code for which a not reachable scenario happen)

File: server/service/individualServices/regardApplicationAutomation.js [hunk 2]
```diff
@@ -220,7 +220,7 @@ async function RequestForInquiringTopologyChangeInformation(applicationName, rel
     let responseCode = response.status;
     if (!responseCode.toString().startsWith("2")) {
```
Comment (PrathibaJee) [inline server/service/individualServices/regardApplicationAutomation.js:221]: if (!responseCode.toString().startsWith("2")) , then you have to check whether the if responseCode.toString() == "408" to confirm that the application is actually not reachable. ( Note : Include the response code for which a not reachable scenario happen)

**Review by PrathibaJee — CHANGES_REQUESTED** [review]

File: server/service/individualServices/regardApplicationAutomation.js (hunk 1 above)
Comment (sunil-kumar-meena) [reply server/service/individualServices/regardApplicationAutomation.js]: done

File: server/service/individualServices/regardApplicationAutomation.js (hunk 2 above)
Comment (sunil-kumar-meena) [reply server/service/individualServices/regardApplicationAutomation.js:222]: done

**Review by PrathibaJee — APPROVED** [review]

**Pull request merged**